src/main/java/pl/edukacyjni/user/UserService.java

```diff
@@ -20,4 +20,11 @@ public class UserService {
     public List<User> getAllUsers() {
         return USERS;
     }
+
+    public User getUserById(long id) {
+        return USERS
+                .stream()
+                .filter(user -> id == user.getId())
+                .findFirst().orElse(null);
```

Comment:
I would not return null in orElse. It's better in that case to throw some Exception here instead of doing that in controller